src/Components/Shared/src/ArrayBuilder.cs

```diff
@@ -6,7 +6,9 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 
-#if COMPONENTS_SERVER
+#if IGNITOR
+namespace Ignitor
+#elif COMPONENTS_SERVER
```

Comment:
`Components.Server.Test` has IVT access to both Components and Components.Server, so we're still forced to have this in a separate namespace.